procdiscovery/pkg/process/process.go

```diff
@@ -3,19 +3,25 @@ package process
 import (
 	"bufio"
 	"fmt"
+	"github.com/odigos-io/odigos/common/envs"
 	"io"
 	"os"
 	"strings"
 
-	"github.com/odigos-io/odigos/common/envOverwrite"
+	"github.com/odigos-io/odigos/common/envs/envOverwrite"
 )
 
 type Details struct {
-	ProcessID int
-	ExeName   string
-	CmdLine   string
-	// Envs only contains the environment variables that we are interested in
-	Envs map[string]string
+	ProcessID    int
+	ExeName      string
+	CmdLine      string
+	Environments *ProcessEnvs
+}
+
+type ProcessEnvs struct {
+	DetailedEnvs map[string]string
+	// OverwriteEnvs only contains the environment variables that we are interested in
```

Comment:
should this comment be updated?

Reply:
I did, it's relevant only for the OverwriteEnvs - to what would you change?

Reply:
```suggestion
	// OverwriteEnvs only contains environment variables that odigos is using for auto-instrumentation and may need to be overwritten
```